tests/sqlparser_common.rs

```diff
@@ -1753,18 +1753,72 @@ fn parse_extract() {
     verified_stmt("SELECT EXTRACT(ISODOW FROM d)");
     verified_stmt("SELECT EXTRACT(ISOYEAR FROM d)");
     verified_stmt("SELECT EXTRACT(JULIAN FROM d)");
+    verified_stmt("SELECT EXTRACT(MICROSECOND FROM d)");
     verified_stmt("SELECT EXTRACT(MICROSECONDS FROM d)");
     verified_stmt("SELECT EXTRACT(MILLENIUM FROM d)");
     verified_stmt("SELECT EXTRACT(MILLENNIUM FROM d)");
+    verified_stmt("SELECT EXTRACT(MILLISECOND FROM d)");
     verified_stmt("SELECT EXTRACT(MILLISECONDS FROM d)");
     verified_stmt("SELECT EXTRACT(QUARTER FROM d)");
     verified_stmt("SELECT EXTRACT(TIMEZONE FROM d)");
     verified_stmt("SELECT EXTRACT(TIMEZONE_HOUR FROM d)");
     verified_stmt("SELECT EXTRACT(TIMEZONE_MINUTE FROM d)");
 
-    let res = parse_sql_statements("SELECT EXTRACT(MILLISECOND FROM d)");
+    let res = parse_sql_statements("SELECT EXTRACT(JIFFY FROM d)");
     assert_eq!(
-        ParserError::ParserError("Expected date/time field, found: MILLISECOND".to_string()),
+        ParserError::ParserError("Expected date/time field, found: JIFFY".to_string()),
```

Comment:
This existing check implies that not having the singular `Millisecond` and `Microsecond` (as opposed to the plural) was intentional, and I was wondering the reason behind this?

Let me know if you'd like me to revert these changes to support only the plural versions.